RebelAllianceBank/Classes/FileHandler.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
This already exist in the global scope of this project

Reply:
fixed